src/analyses/base.ml

```diff
@@ -1317,7 +1317,7 @@ struct
               | PlusA|PlusPI|IndexPI -> checkBinop e1 e2 (GobOption.map2 Z.(+))
               | MinusA|MinusPI|MinusPP -> checkBinop e1 e2 (GobOption.map2 Z.(-))
               | Mult -> checkBinop e1 e2 (GobOption.map2 Z.mul)
-              | Div -> checkBinop e1 e2 (GobOption.map2 Z.div)
+              | Div -> (try checkBinop e1 e2 (GobOption.map2 Z.div) with Division_by_zero -> true)
```

Comment:
It seems to me that this only returns `true` if one of the bounds of the interval for `e2` is equal to zero. Should we also return `true` if the interval for `e2` contains zero?

Reply:
This is a valid point, since this may have an influence on the monotonicity of other functions, would it not? Is there another way to achieve this without specializing ```checkBinop```?